tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types.ts

```diff
@@ -123,6 +123,88 @@ export interface GraphExecution extends GraphExecutionDigest {
   device_name: string;
 }
 
+/**
+ * Information about an op in a TensorFlow graph.
+ *
+ * Including its enclosing graph, relation with other ops in the graph
+ * (inputs and consumers), and the source-code location (stack trace)
+ * at which the op was created.
+ */
+export interface GraphOpInfo {
```

Comment:
This data structure is quite similar to graphdef and graph plugin's internal data structure.
I am a bit weary of introducing yet another data structure without getting Davi'd buy-in.